subgraph-js/src/schema-helper/buildSchemaFromSDL.ts

```diff
@@ -165,19 +165,17 @@ export function addResolversToSchema(
     const fieldMap = type.getFields();
 
     for (const [fieldName, fieldConfig] of Object.entries(fieldConfigs)) {
-      if (fieldName === "__resolveReference") {
+      if (fieldName === '__resolveReference') {
         type.extensions = {
           ...type.extensions,
           apollo: {
             ...type.extensions.apollo,
             subgraph: {
               ...type.extensions.apollo?.subgraph,
               resolveReference: fieldConfig,
-            }
+            },
           },
         };
-      } else if (fieldName.startsWith("__")) {
-        (type as any)[fieldName.substring(2)] = fieldConfig;
         continue;
```

Comment:
Note to reviewer: I restored this `continue` in the `resolveReference` case, which previously existed here: https://github.com/apollographql/federation/pull/1746/files#diff-89b6c52a33464884610c771bf9c04484fc8acf56fb57fd9a31b168dd55595530R181

Bit tough to see with the diff. No effect on tests with or without.

Reply:
Good catch — sorry for missing that in the last review. I guess technically a no-op since `fieldMap['__resolveReference']` shouldn't be truthy.